core/src/main/java/com/predic8/membrane/core/interceptor/MessageAnalyser.java

```diff
@@ -104,11 +113,16 @@ public Outcome handleRequest(Exchange exc) throws Exception {
 	}
 
 	@Override
-	public Outcome handleResponse(Exchange exc) throws Exception {
+	public Outcome handleResponse(Exchange exc) {
 
-		ExtractedData data = analyse(exc.getResponse());
+        ExtractedData data = null;
+        try {
+            data = analyse(exc.getResponse());
+        } catch (Exception e) {
+            log.error("Could not analyse response: {}",e.getMessage(), e);
```

Comment:
`e.getMessage()` is already printed, because `e` is passed to `log.error(...)` in the `Throwable` parameter.